_spinnaker_install_admin_guides/secrets-s3.md

```diff
@@ -1,28 +1,19 @@
 ---
 layout: post
-title: Spinnaker Secrets and S3
-order: 150
+title: Secrets with S3
+order: 151
 ---
-Managing Spinnaker secrets separately from its configuration is a necessary step to enabling Spinnaker through an SCM like git. This document describes how to store secrets in s3 and these secrets are used in Spinnaker and Halyard.
 
 {:toc}
 
-## Overview
-We can now store secrets (tokens, passwords, sensitive files) separately from the Spinnaker and Halyard configurations as of `halyard-armory:1.4.1`. We'll provide references to these secrets to services that need them.
+This document describes how to set up Spinnaker secrets in an encrypted S3 bucket. In this example, we'll be using a `mybucket` bucket in the `us-west-2` region to store Github credentials and a kubeconfig file. We'll be referencing the bucket by its URL `mybucket.us-west-2.amazonaws.com`.
 
-- Spinnaker services that support decryption will decrypt these secrets upon startup.
-- Halyard can decrypt these secrets when it needs to use them (e.g. when validating resources).
-- Halyard can send secret references to the services that support decryption or send decrypted secrets if the service does not support it.
-
-## Secrets in S3
-In this example, we'll be using a `mybucket` bucket in the `us-west-2` region to store Github credentials and a kubeconfig file. We'll be referencing the bucket by its URL `mybucket.us-west-2.amazonaws.com`.
-
-### Authorization
+## Authorization
 Since we're storing sensitive information, we'll protect the bucket by restricting access and [enabling encryption](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/default-bucket-encryption.html).
 
-The important thing to remember is to run Halyard's daemon and Spinnaker services that support decryption with IAM roles that allows them to read that content.
-
+The important thing to remember is to run Halyard's daemon and Spinnaker services that support decryption with IAM roles that allow them to read that content.
 
+## Storing secrets
 ### Storing credentials
 Let's store our github credentials in `mybucket/spinnaker-secrets.yml`:
 
```

Comment:
Is a triple backtick missing?

Reply:
This was just trying to organize the topics and subtopics. Storing secrets in general is the topic, storing credentials and storing files are the subtopic. The ## header also automatically gives you the nice horizontal dividing line. It just looked more organized.